runtime/lua/vim/lsp/buf.lua

```diff
@@ -23,6 +23,17 @@ local function request(method, params, callback)
   return vim.lsp.buf_request(0, method, params, callback)
 end
 
+local function notify(method, params)
+  validate {
+    method = {method, 's'};
+  }
+  return vim.lsp.buf_notify(0, method, params)
+end
```

Comment:
what's the point of this function instead of calling `vim.lsp.buf_notify` directly?

Reply:
Just wanted to be consistent with the `request` method. Will call `buf_notify` directly and remove `notify` function